services/storage/src/simcore_service_storage/exceptions.py

```diff
@@ -37,16 +42,13 @@ class LinkAlreadyExistsError(DatabaseAccessError):
     msg_template: str = "The link {file_id} already exists"
 
 
-class S3AccessError(StorageRuntimeError):
-    code = "s3_access.error"
-    msg_template: str = "Unexpected error while accessing S3 backend"
+assert S3AccessError  # nosec
+assert S3BucketInvalidError  # nosec
+assert S3KeyNotFoundError  # nosec
 
 
-class S3BucketInvalidError(S3AccessError):
-    code = "s3_bucket.invalid_error"
-    msg_template: str = "The bucket '{bucket}' is invalid"
-
-
-class S3KeyNotFoundError(S3AccessError):
-    code = "s3_key.not_found_error"
-    msg_template: str = "The file {key}  in {bucket} was not found"
+__all__: tuple[str, ...] = (
+    "S3AccessError",
+    "S3BucketInvalidError",
+    "S3KeyNotFoundError",
+)
```

Comment:
why not import them from the library where they are defined? This is just a way to proxy the imports. And the tooling in general wants us to avoid it.

Reply:
Because this is an exception module inside a Storage service, I do not see a problem with that. Yes, I moved those 3 exceptions from this module to our AWS library, but still there are more exceptions in this module, I don't see a huge benefit in modifying all of the code everywhere when there is an exception module, or?

Reply:
Fixed